### DISCUSSION TITLE
detailed-docs.md seems to be missing

### DISCUSSION THREAD

**Discussion by MarkusWillinger** [actions/actions-runner-controller#2139] [General]
Hello,

in the [Quickstart document,](https://github.com/actions/actions-runner-controller/blob/master/docs/quickstart.md) when clicking "[Detailed Documentation](https://github.com/actions/actions-runner-controller/blob/master/docs/detailed-docs.md)", I get the following 404 error:
The 'actions/actions-runner-controller' repository doesn't contain the 'docs/detailed-docs.md' path in 'master'.

**Comment by Link-** [comment]
Thank you @MarkusWillinger for reporting this. We recently overhauled the docs and this must have slipped through the cracks. 

This section in the README lists everything that was in the detailed documentation split into dedicated sections:

https://github.com/actions/actions-runner-controller/blob/master/README.md#documentation

**Reply by Link-** [reply]
I will create a PR to fix the broken link

**Reply by MarkusWillinger** [reply]
Thanks a lot for your help!
